pronto_core/src/lidar_odometry_module.cpp

```diff
@@ -73,6 +73,7 @@ Update* LidarOdometryModule::processMessage(const LidarOdometryUpdate *msg,
     // at this point, the mode is either position or position_orient
     if (mode_ == LidarOdometryMode::POSITION) {
       z_meas.head<3>() = t1_body_vo_.translation();
+      // z_meas.head<3>() = t0_body_filter_.translation() + msg->relative_pose.translation();
```

Comment:
I think this is the correct way to get relative translation as opposed to the current method which applies a rotation

Reply:
I'm not entirely sure...probably it doesn't make any difference since the rotational part of the measurement would be identity anyway?